docassemble/AssemblyLine/data/questions/al_saved_sessions.yml

```diff
@@ -1,267 +1,151 @@
+  interview_list(action="delete", filename=action_argument("filename"), session=action_argument("session"))
+  log("Answer set deleted", "success")
 ---
-need:
-  - file_like
-# For save snapshot
-code: |
-  filtered_vars_tmp = all_variables(simplify=False)
-  simple_filtered_vars_tmp = all_variables()
+id: load answer
+decoration: folder-open
+question: |
+  Load answer set
+subquestion: |
+  % if not user_logged_in():
+  <div class="alert alert-danger fade show d-flex" role="alert">
+  :exclamation-triangle: &nbsp;
+  <div>
+  You need to be signed in to save an answer set.
+  </div>
+  </div>
   
-  for var in al_sessions_variables_to_remove:
-    filtered_vars_tmp.pop(var, None)
-    simple_filtered_vars_tmp.pop(var, None)   
+  The "answer set" feature lets you make a copy of your answers to
+  use again and again. You can load this answer set in a new interview later.
   
-  al_simple_filtered_vars = simple_filtered_vars_tmp
+  ${ action_button_html(url_of('login'), label="Sign in", icon="sign-in-alt", size="md", color="primary") }
+  % else:
+  % if not interview_list_html():
+  You haven't created any answer sets yet. 
   
-  # Delete all files and ALDocuments
-  al_sessions_filtered_vars = {item:filtered_vars_tmp[item] for item in filtered_vars_tmp if not file_like(filtered_vars_tmp[item])}
-  del filtered_vars_tmp
-  del simple_filtered_vars_tmp
----
-# For fast forward
+  You can
+  <a href="${ url_of('interviews') }" class="alert-link">visit in-progress forms</a>
+  to keep working on a form that you started earlier. To create an answer set, use the
+  "Save answer set" link in the main menu.
+  % else:
+  Tap the name of an answer set below to re-use the answers in the form you are working
+  on now. You can
+  <a href="${ url_of('interviews') }" class="alert-link">visit in-progress forms</a>
+  to keep working on a form that you started earlier.
+
+  ${ interview_list_html() }
+  % endif
+  % endif
+event:  al_load_saved_session_screen
+back button label: Back
 ---
-need:
-  - file_like
+#reconsider:
+#  - al_load_saved_session_screen
+event: al_load_saved_session
 code: |
-  # vars = get_session_variables(filename, session_id, secret, simplify=False)
-  al_sessions_source_session = int(al_sessions_source_session)
-  filtered_vars_tmp = get_session_variables(
-    al_sessions_list[al_sessions_source_session].get('filename'),
-    al_sessions_list[al_sessions_source_session].get('session'),
-    simplify=False)
-   
-  for var in al_sessions_variables_to_remove:
-    filtered_vars_tmp.pop(var, None)
-
-  # Delete all files and ALDocuments
-  al_sessions_fast_forward_filtered_vars = {item:filtered_vars_tmp[item] for item in filtered_vars_tmp if not file_like(filtered_vars_tmp[item])}
-        
-  del filtered_vars_tmp
+  # Using this code block that then loads a screen is critical to get this to work
+  # without taking people back to the screen after they load the answers
+  # set_save_status('ignore')
```

Comment:
Looks like this function was removed at some point?

Reply:
`set_save_status('ignore')` appeared to surface some confusingness or Docassemble bug. It would be nice for saving the interview to not add a "Step" in the progress bar but it's not crucial.

Should probably remove the commented out code anyway.